sklearn/ensemble/stacking_classifier.py

```diff
@@ -0,0 +1,295 @@
+import numpy as np
+from ..base import BaseEstimator, ClassifierMixin
+from ..base import clone, is_classifier
+from ..model_selection._validation import cross_val_predict
+from ..preprocessing import LabelEncoder
+from ..externals.joblib import Parallel, delayed
+from ..utils.metaestimators import if_delegate_has_method
+from ..utils.validation import has_fit_parameter, check_is_fitted
+from ..externals import six
+
+
+def _parallel_predict(estimator, method, X, y, fit_params):
+    """Private function used to fit an estimator within a job."""
+    estimator.fit(X, y, **fit_params)
+    return estimator, getattr(estimator, method)(X)
+
+
+class StackingClassifier(BaseEstimator, ClassifierMixin):
+    """ Stacking classifier for combining unfitted estimators
```

Comment:
I know what you mean by unfitted but I feel it is a bit awkward here and in the next sentence.